Application/EdFi.Ods.AdminApi/appsettings.json

```diff
@@ -8,6 +8,9 @@
         "PreventDuplicateApplications": false,
         "EnableAdminConsoleAPI": false
     },
+    "CorsSettings": {
```

Comment:
It seems that this is only apply for the "Admin Console" app. What if we create a section called "AdminConsole" in the appsettings?

Also we have to add the origins property here in case we are in a qa/production environment.